BaseFilter/sections/antiadblock.txt

```diff
@@ -805,6 +805,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/141887
+||exchangedetail.com^
```

Comment:
It's Admiral. So, what you need to do:
1. Remove your rule
2. Add `pwinsider.com` to the Admiral rules in AdGuard Annoyances 
```adblock
/^https?:\/\/[^\/]+\.[^\/]+\/[a-zA-Z0-9_-]{32,}$/$script,third-party,domain=
! cannot be closed
! #%#//scriptlet("abort-current-inline-script", "document.createElement", "admiral")
```
3. Add `exchangedetail.com` to the same section and to AdGuard Tracking Protection. You'll see the note `! Add same rules to the Admiral section of Tracking Protection filter`.

Reply:
Fixed.